src/kbmod/analysis_utils.py

```diff
@@ -14,8 +14,8 @@
 from sklearn.cluster import DBSCAN, OPTICS
 
 from .file_utils import *
+from .filters.stats_filters import *
```

Comment:
I think that it's permissible to import `*` in ``__init__.py`` files because it's a significant hassle otherwise, and I do understand we import `*` everywhere in our code, but I really don't think we should continue doing that forward. 

It's ok not to change this right, I think that at some point I will go and redo all of the files at the same time.